### PR TITLE
New ClickHouse connector

### PR DESCRIPTION
**Pull request by csansoon**
Right now we rely on clickhouse's postgres interface to connect to Clickhouse with our postgres client. This is far from ideal specially considering Clickhouse's own cloud offering does not allow you to activate the postgres inteface. We need to reimplement the connector with Clickhouse's offical [nodejs client](https://clickhouse.com/docs/en/integrations/language-clients/javascript).

WIP, still testing it

### CODE REVIEW

File: packages/connectors/clickhouse/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      return this.convertDataType(nullableMatch[1]!)
+    }
+
+    if (dataTypeID in ['JSON', 'UUID', 'IPv4', 'IPv6', 'Enum8', 'Enum16', 'Enum']) {
+      return DataType.String
+    }
+
+    return fallbackType
+  }
+}
```
Comment (csansoon) [inline packages/connectors/clickhouse/src/index.ts:167]: There's no `DataType` type in ClickHouse's module...

File: packages/connectors/clickhouse/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - @latitude-data/query_result@0.0.2
+
+## 0.0.1
+
+### Patch Changes
+
+- First published release of Latitude data!
+- Updated dependencies
+  - @latitude-data/base-connector@0.0.1
+  - @latitude-data/query_result@0.0.1
```
Comment (andresgutgon) [inline packages/connectors/clickhouse/CHANGELOG.md:61]: What's all of this?

Comment (csansoon) [reply packages/connectors/clickhouse/CHANGELOG.md:61]: leftovers :o

File: packages/connectors/clickhouse/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@latitude-data/clickhouse-connector",
+  "version": "1.1.2",
+  "license": "LGPL",
```
Comment (andresgutgon) [inline packages/connectors/clickhouse/package.json:4]: 👀

File: packages/connectors/clickhouse/package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@latitude-data/clickhouse-connector",
+  "version": "1.1.2",
```
Comment (andresgutgon) [inline packages/connectors/clickhouse/package.json:3]: Start in `0.0.0` so when doing changeset minor it goes to `0.1.0`

**Review by andresgutgon — APPROVED** [review]

**Pull request merged**